### PR TITLE
security policies

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview currently here: https://docs.preview.docs.r3.com/en/platform/corda/5.0/deploying-operating/config/security-policies.html

### CODE REVIEW

File: content/en/platform/corda/5.0/deploying-operating/config/security-policies.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ section_menu: corda5
...
+* `basic_security.policy` — a basic security policy that prevents only the most critical security risks.
+* `medium_security.policy` —  a medium security policy that prevents the most critical security risks and additionally prevents access to files and the network.
+
 Policies are based on [Conditional Permissions](https://docs.osgi.org/specification/osgi.core/8.0.0/service.condpermadmin.html#i1534586) of the OSGi security model. Policies can have `ALLOW` and `DENY` access blocks that represent a string encoded [ConditionalPermissionInfo](https://docs.osgi.org/javadoc/r4v42/org/osgi/service/condpermadmin/ConditionalPermissionInfo.html#getEncoded()). A block at a higher position has a higher priority. Each block starts with conditions that must be satisfied in order to apply that block. Next, the block lists a sect of permissions that are either allowed or denied based on the block type. The [basic syntax](https://docs.osgi.org/specification/osgi.core/8.0.0/service.condpermadmin.html#i1716478) is:
```
Comment (mbrkic-r3) [inline content/en/platform/corda/5.0/deploying-operating/config/security-policies.md:22]: > `sect` of permissions

Is this right?

Comment (nadinequinr3) [reply content/en/platform/corda/5.0/deploying-operating/config/security-policies.md]: Resolved. Thank you!

**Review by mbrkic-r3 — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**